pynucastro/reduction/reduction.py

```diff
@@ -45,29 +46,22 @@ def get_net_info(net, comp, rho, T):
     ebind = np.zeros(len(net.unique_nuclei), dtype=np.float64)
     m = np.zeros(len(net.unique_nuclei), dtype=np.float64)
 
-    mass_neutron = 1.67492721184e-24
-    mass_proton = 1.67262163783e-24
-    c_light = 2.99792458e10
-
     for i, n in enumerate(net.unique_nuclei):
 
         y[i] = y_dict[n]
         ydot[i] = ydots_dict[n]
         z[i] = n.Z
         a[i] = n.A
         ebind[i] = n.nucbind or 0.0
-        m[i] = mass_proton * n.Z + mass_neutron * n.N - ebind[i] / c_light**2
```

Comment:
The units here look wrong. `mass_proton` and `mass_neutron` are in grams, but `ebind[i]` is in MeV/nucleon. We should probably just replace it with `n.A_nuc * constants.m_u`.

Reply:
yeah, I think we should switch this to use `n.mass`

Reply:
fixed